rocq-core.opam

```diff
@@ -0,0 +1,50 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+version: "dev"
+synopsis: "The Coq Proof Assistant -- prelude"
```

Comment:
It's up to us how to describe this package, but I would tend to say that this is the package of the Rocq prover runtime + its prelude (given that we will consider `rocq-runtime` as an internal package that may be removed sometimes in the future and that users should not directly depend on).